lua/autorun/cl_streamcore.lua

```diff
@@ -99,8 +113,8 @@ hook.Add( "Think", "XTS_SC_Think", function( ent )
         if IsValid( station ) and IsValid( ent ) and IsValid( from ) then
             if streamtbl[6] then
                 local distance = LocalPlayer():GetPos():Distance( ent:GetPos() )
-                distance = math.Clamp( (distance-streamtbl[8] )/30, 1, 300 )
-                local volume = streamtbl[7]/distance
+                distance = math.Clamp( (distance-streamtbl[8] ) / 30, 1, 300 )
+                local volume = streamtbl[7] / distance
```

Comment:
may as well fix all the operators here if youre going to fix  some of them
```suggestion
                distance = math.Clamp( ( distance - streamtbl[8] ) / 30, 1, 300 )
                local volume = streamtbl[7] / distance
```